scripts/target.js

```diff
@@ -0,0 +1,281 @@
+const SESSION_COOKIE_NAME = 'sessionId';
+const SESSION_COOKIE_EXPIRATION_DAYS = 7;
+
+/**
+ * Escape a selector.
+ * @param selector
+ * @returns {string}
+ */
+function escapeSelector(selector) {
+  return selector.replaceAll(/#(\d)/g, '#\\3$1 ');
+}
+
+/**
+ * Generate a UUID.
+ * @returns {string}
+ */
+function uuid() {
+  return `sess-${Math.random()
+    .toString(36)
+    .substring(2, 9)}-${Date.now()
+    .toString(36)}`;
+}
+
+/**
+ * Get a cookie by name.
+ * @param name
+ * @returns {string}
+ */
+function getCookie(name) {
```

Comment:
Do we actually need to use cookies for target to work properly? I wonder if we could leverage one of the Store APIs instead and save a few bytes on every TCP packet